src/components/menu/menu.stories.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Menu from './menu.svelte'
+  import Icon from '../icon/icon.svelte'
+
+  let button
+</script>
+
+<Meta title="Menu" component={Menu} />
```

Comment:
Add `argTypes` prop and set the defaultValue for `isOpen` to `true` so the menu is open by default.

Reply:
See the `label` prop in `input.stories.svelte`

Reply:
done